2024-12-10 21:24:58,590 - INFO - Finished! Total sfe time: 10.542316675186157 
2024-12-10 21:24:58,590 - DEBUG - 237,server, Parsed server ram usage: 80644
2024-12-10 21:24:58,591 - DEBUG - 237,client, Parsed client ram usage: 80616
2024-12-10 21:25:20,875 - DEBUG - client all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 11 second(s) each.', '2024-12-10 21:23:30,332 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.1:61002', '2024-12-10 21:23:30,335 - tno.mpc.communication.httphandlers - ERROR - Message not received.', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1061, in _wrap_create_connection', '    sock = await aiohappyeyeballs.start_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 103, in start_connection', '    raise first_exception', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 81, in start_connection', '    sock = await _connect_sock(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 165, in _connect_sock', '    await loop.sock_connect(sock, address)', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 502, in sock_connect', '    return await fut', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 537, in _sock_connect_cb', "    raise OSError(err, f'Connect call failed {address}')", "ConnectionRefusedError: [Errno 111] Connect call failed ('192.168.1.2', 61001)", '', 'The above exception was the direct cause of the following exception:', '', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/tno/mpc/communication/httphandlers.py", line 193, in _send', '    async with self.session.post(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 1355, in __aenter__', '    self._resp: _RetType = await self._coro', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 659, in _request', '    conn = await self._connector.connect(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 557, in connect', '    proto = await self._create_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1002, in _create_connection', '    _, proto = await self._create_direct_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1336, in _create_direct_connection', '    raise last_exc', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1305, in _create_direct_connection', '    transp, proto = await self._wrap_create_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1076, in _wrap_create_connection', '    raise client_error(req.connection_key, exc) from exc', "aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host 192.168.1.2:61001 ssl:default [Connect call failed ('192.168.1.2', 61001)]", '2024-12-10 21:23:31,399 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-10 21:23:31,432 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-10 21:23:31,522 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-10 21:23:31,574 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', 'TP is  0.001', '2024-12-10 21:23:31,591 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-10 21:23:31,613 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-10 21:23:31,614 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.1:61002 shutdown', 'Total bytes received: 1930', 'Total messages received: 5', '2024-12-10 21:23:31,618 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.2:61001 shutdown', 'Total bytes sent: 766', 'Total messages sent: 5', '2024-12-10 21:23:31,619 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 766', 'Total messages sent: 5', 'Total bytes received: 1930', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 1 237"', '\tUser time (seconds): 6.61', '\tSystem time (seconds): 0.78', '\tPercent of CPU this job got: 86%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:08.51', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80328', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 16069', '\tVoluntary context switches: 63', '\tInvoluntary context switches: 203', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_10mbits_2024-12-10_21-24-48.csv']
2024-12-10 21:25:20,875 - DEBUG - server all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 11 second(s) each.', '2024-12-10 21:24:19,944 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.2:61001', '2024-12-10 21:24:20,975 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-10 21:24:21,088 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-10 21:24:21,116 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-10 21:24:21,169 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', 'TP is  0.001', '2024-12-10 21:24:21,215 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-10 21:24:21,220 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-10 21:24:21,222 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.2:61001 shutdown', 'Total bytes received: 766', 'Total messages received: 5', '2024-12-10 21:24:21,225 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.1:61002 shutdown', 'Total bytes sent: 1930', 'Total messages sent: 5', '2024-12-10 21:24:21,226 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 1930', 'Total messages sent: 5', 'Total bytes received: 766', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 0 237"', '\tUser time (seconds): 6.65', '\tSystem time (seconds): 0.87', '\tPercent of CPU this job got: 87%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:08.64', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80576', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 15969', '\tVoluntary context switches: 65', '\tInvoluntary context switches: 118', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_10mbits_2024-12-10_21-24-48.csv']
2024-12-10 21:25:22,327 - DEBUG - All values from powertop for client: 1       140 mW  
18     5.33 mW  
22     3.69 mW  
Name: 7, dtype: object
2024-12-10 21:25:22,327 - DEBUG - Energy client: 1.1012578
2024-12-10 21:25:23,665 - DEBUG - All values from powertop for server: 1       529 mW  
28     1.15 mW  
41      521 uW  
Name: 7, dtype: object
2024-12-10 21:25:23,665 - DEBUG - Energy server: 3.94819224
